iroha_p2p/src/peer.rs

```diff
@@ -485,113 +510,3 @@ impl Garbage {
         Ok(())
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
Why do you think we should remove these tests?

Reply:
Moreover I think now that we discussed this possible case. We should add a test that checks a case when one peer turns offline and then successfully reconnects and is able to send messages again.

Reply:
These tests don't work in this PR because peers don't work without network actor.

Reply:
What about the test from my second suggestion?

Reply:
This behavior (along the tween elimination) will be possible in next PR.
Now we just drop the peer that is disconnected, and if it connects to us it will be like another new peer. (We have no identification now, will have in next PR).